Bindings/Java/OpenSimJNI/CMakeLists.txt

```diff
@@ -30,21 +32,44 @@ add_custom_command(
             "${CMAKE_SOURCE_DIR}/Bindings/OpenSimHeaders_actuators.h"
             "${CMAKE_SOURCE_DIR}/Bindings/OpenSimHeaders_analyses.h"
             "${CMAKE_SOURCE_DIR}/Bindings/OpenSimHeaders_tools.h"
-    COMMENT "Generating java bindings source code with SWIG."
```

Comment:
These dependencies are incorrect, I think. Generating the wrapper for `java_simbody` doesn't depend on `OpenSimHeaders_tools.h`. See how I handle this for python: https://github.com/opensim-org/opensim-core/blob/master/Bindings/Python/CMakeLists.txt#L188.